script/release-artifact-cleanup.js

```diff
@@ -90,9 +90,9 @@ async function deleteTag (tag, targetRepo) {
 
 async function cleanReleaseArtifacts () {
   const tag = args.tag
-  const lastBumpCommit = getLastBumpCommit().message
+  const isNightly = args.tag.includes('nightly')
```

Comment:
We could have used local `tag` here instead if `args.tag` =/ Doesn't really matter though.